installer.cmd

```diff
@@ -0,0 +1,257 @@
+::@Kimbyeongjang - (김병장#7165)
+@echo off
+title A32NX Builder
+
+>nul 2>&1 "%SYSTEMROOT%\system32\cacls.exe" "%SYSTEMROOT%\system32\config\system"
+if '%errorlevel%' NEQ '0' (
+    echo Request Administrator Privilege ...
+    goto UACPrompt
+) else ( goto gotAdmin )
+
+:UACPrompt
+    echo Set UAC = CreateObject^("Shell.Application"^) > "%temp%\getadmin.vbs"
+    set params = %*:"=""
+    echo UAC.ShellExecute "cmd.exe", "/c %~s0 %params%", "", "runas", 1 >> "%temp%\getadmin.vbs"
+    "%temp%\getadmin.vbs"
+    rem del "%temp%\getadmin.vbs"
+    exit /B

```

Comment:
```suggestion
    exit
```

No need for `/B` as we're [not setting an error code](https://ss64.com/nt/exit.html).